metaconfig-core/shared/src/main/scala/metaconfig/Configured.scala

```diff
@@ -51,11 +51,11 @@ sealed abstract class Configured[+A] extends Product with Serializable {
 
 trait ConfiguredLowPriorityImplicits {
 
-  implicit def toOption[A](value: Configured[_ <: A]): Option[A] =
-    value match {
-      case Configured.Ok(v) => Some(v)
-      case _ => None
-    }
+  // implicit def toOption[A](value: Configured[A]): Option[A] =
```

Comment:
TODO: restore, I was incorrectly blaming it for weird compilation errors. That said, it will look differently in Scala 3

Reply:
May be this shouldn't be restored at all? It seemed to marry covariance with implicit conversion. How much is it used?